README.md

```diff
@@ -47,6 +47,32 @@ Documentation chapters:
 - [Writing middlewares](docs/writing_middleware.md)
 - [How to make multiple commands atomic](docs/multiple_commands.md)
 
+## Admin web UI
+
+`pg_eventstore` implements admin UI where you can browse various database objects. It is implemented as rack application. It doesn't have any authentication/authorization mechanism - it is your responsibility to take care of it.
```

Comment:
In rails this can be done by implementing route constraints. Here is an example how to do it using devise gem https://github.com/heartcombo/devise/wiki/How-To%3A-Protect-Resque-Web-with-Devise.

Another approach is to setup basic auth for the particular url on the nginx level.